tests/test_class_attributes.rs

```diff
@@ -0,0 +1,29 @@
+use pyo3::prelude::*;
+
+mod common;
+
+#[pyclass]
+struct Foo { }
```

Comment:
rustfmt complains about it.
```suggestion
struct Foo {}
```